Makefile.linux

```diff
@@ -97,18 +97,23 @@ endif
 
 BUILD_OBJ = $(CORE_OBJ:%.o=$(BUILD_DIR)/%.o)
 
-LINUX_MODULES="zjs_buffer.json,\
+# define the modules that will be pulled into the linux build
+LINUX_MODULES="zjs_board.json,\
+zjs_buffer.json,\
 zjs_console.json,\
-zjs_event.json,
+zjs_event.json,\
+zjs_gpio.json,
```

Comment:
This pulls in GPIO mock magically; I see that mock.c got built and can run the GPIO tests. Also, Travis does that.